### PR TITLE
Add E-T-L CLI commands

### PR DESCRIPTION
**Pull request by andreibancioiu**
Here we add the CLI commands for the extract & load jobs. The actual implementation of the jobs is contained in the next PR.

### CODE REVIEW

File: multiversxetl/cli/work.py [hunk 1]
```diff
@@ -0,0 +1,228 @@
+import logging
+import socket
+import time
+from pathlib import Path
+from typing import Callable, Optional
+
+import click
+
+from multiversxetl.errors import TransientError
+from multiversxetl.jobs import ExtractJob, FileStorage, LoadJob, TransformJob
```
Comment (andreibancioiu) [inline multiversxetl/cli/work.py:10]: Implementation will follow in the next PR.

File: multiversxetl/cli/work.py [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+        workspace: str,
+        gcp_project_id: str,
+        worker_id: str,
+        index_name: Optional[str],
+        continue_on_error: bool,
+        sleep_between_tasks: int
+):
+    """
+    This command runs continuously (until interrupted) and extracts **time-stamped indices** from Elasticsearch, 
+    then saves it (as JSON files) in the `workspace` folder.
```
Comment (bogdan-rosianu) [inline multiversxetl/cli/work.py:35]: saves them

Comment (schimih) [inline multiversxetl/cli/work.py:35]:
```suggestion
    then saves them (as JSON files) in the `workspace` folder.
```

**Review by schimih — COMMENTED** [review]
only typo

File: multiversxetl/cli/work.py [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+        logger.log_info(f"Starting transform & load, index = {task.index_name}, task = {task.id} ...", data=task.to_dict())
+
+        transform_job = TransformJob(file_storage, task)
+        transform_job.run()
+        load_job = LoadJob(gcp_project_id, file_storage, task, Path(schema_folder))
+        load_job.run()
+        storage.update_task(task.id, lambda t: t.update_on_loading_finished(""))
+        file_storage.remove_transformed_file(task.get_pretty_name())
+
+        logger.log_info(f"Transform & load finished, index = {task.index_name}, task = {task.id},", data=task.to_dict())
```
[the remark this answers is not in the document]
Comment (andreibancioiu) [reply multiversxetl/cli/work.py:206]: Fixed.

File: multiversxetl/cli/work.py (hunk 2 above)
Comment (andreibancioiu) [reply multiversxetl/cli/work.py:35]: Fixed.

Comment (andreibancioiu) [reply multiversxetl/cli/work.py:35]: Fixed.

**Review by schimih — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**